include/rdma/fi_eq.h

```diff
@@ -73,9 +71,12 @@ struct fid_wait {
 	struct fi_ops_wait	*ops;
 };
 
+struct pthread_mutex_t;
+struct pthread_cond_t;
```

Comment:
non-standard? sure? i tested such construct on all available compilers (icc/gcc/MS) - it works everywhere...
i will revert these changes